source/MRMesh/MRMeshLoad.cpp

```diff
@@ -20,16 +21,16 @@ namespace MR
 namespace MeshLoad
 {
 
-tl::expected<Mesh, std::string> fromMrmesh( const std::filesystem::path & file, Vector<Color, VertId>* )
+tl::expected<Mesh, std::string> fromMrmesh( const std::filesystem::path& file, Vector<Color, VertId>*, ProgressCallback callback )
 {
     std::ifstream in( file, std::ifstream::binary );
     if ( !in )
         return tl::make_unexpected( std::string( "Cannot open file for reading " ) + utf8string( file ) );
 
-    return fromMrmesh( in );
+    return fromMrmesh( in, nullptr, callback );
```

Comment:
why nullptr, pass function arg

Reply:
nullptr because missed argument isn't used (i think it added for interface compatibility with other functions (mb do some rework) )